src/constants/notificationSchema.js

```diff
@@ -22,6 +22,7 @@ notificationSchemaPlaceholder[
     'A_COLLABORATION_WAS_CREATED_ON_YOUR_BEHALF_MORE_DETAILED',
   showNotificationDialog: false,
   buttonPath: '/#collab-card',
+  notificationAvatar: 'userName',
```

Comment:
Maybe something like `notificationAvatarKey` or `notificationAvatarField` is clearer?

Reply:
Done.